package.json

```diff
@@ -61,9 +61,9 @@
     "@atomist/tree-path": "^1.0.3"
   },
   "devDependencies": {
-    "@atomist/automation-client": "2.0.0-master.20191207145203",
+    "@atomist/automation-client": "2.0.0-master.20191213160252",
     "@atomist/microgrammar": "^1.2.1",
-    "@atomist/sdm": "2.0.0-master.20191207100157",
+    "@atomist/sdm": "2.0.0-sdm-mutations.20191213145220",
```

Comment:
Do you want to wait until the sdm PR is merged and this updated to merge this?

Reply:
Just did that. Thanks.